WordPress/WordPressTest/ViewRelated/Views/Controllers/NoResultsViewControllerTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  NoResultsViewControllerTests.swift
+//  WordPressTest
+//
+//  Created by Jack Chen on 2022/9/18.
+//  Copyright © 2022 WordPress. All rights reserved.
+//
+
```

Comment:
Love these tests!

Would you mind wiping the header here so we match the others?

Reply:
Ok, Finished in  33876dc0d336ed3505b78a588fbb420a122b8244